dev/reference/performance/follower-reads.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Follower Reads
+summary: This document describes the use and implementation of Follower Reads.
+category: reference
+---
+
+# Follower Reads
```

Comment:
how about `Follower Read`?

Reply:
@zz-jason Also a choice. Then we can keep the file name of Chinese version and English version consistent.